libjulia/backends/evm/EVMCodeTransform.cpp

```diff
@@ -60,12 +60,14 @@ CodeTransform::CodeTransform(
 	solAssert(deposit == 0, "Invalid stack height at end of block.");
 }
 
-void CodeTransform::operator()(const FunctionDefinition&)
+void CodeTransform::operator()(const dev::solidity::assembly::FunctionDefinition&)
```

Comment:
I don't think the namespaces are required here? If the `ErrorReporter` is including conflicting namespaces, we should fix that.